web/src/lib/hashing.ts

```diff
@@ -0,0 +1,29 @@
+import { AbiEncodedValue, IDKitConfig } from "@worldcoin/idkit-core";
+
+import {
+  HashFunctionOutput,
+  hashToField,
+  packAndEncode,
+  solidityEncode,
+} from "@worldcoin/idkit-core/hashing";
+
+export const validateABILikeEncoding = (value: string): boolean => {
+  const ABI_REGEX = /^0x[\dabcdef]+$/;
+  return !!value.toString().match(ABI_REGEX) && value.length >= 66; // Because `0` contains 66 characters
+};
+
+export const generateExternalNullifier = (
```

Comment:
let's mark this as DEPRECATED. The developer portal will no longer generate and store external nullifiers (upcoming change).